lib/helpers.ts

```diff
@@ -77,3 +82,17 @@ export const bulkWrapWith = (
   paths: string[],
   wrappingFn: Parameters<typeof wrapWith>[2],
 ) => paths.forEach((path: string) => wrapWith(obj, path, wrappingFn))
+
+export function getAuthorizationClaimItems(
+  authInfo: AuthenticationInfo,
+  claim: string,
+  tenant?: string,
+): string[] {
+  let value: unknown
+  if (tenant) {
+    value = authInfo.token[authorizedTenantsClaimName]?.[tenant]?.[claim]
```

Comment:
why are we using dynamic object keys and not static ones?
Also, I see that the property `tenants` does not exist on type `Token`